components/chainhook-sdk/Cargo.toml

```diff
@@ -8,17 +8,17 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-serde = {version = "1", features = ["rc"]}
+serde = { version = "1", features = ["rc"] }
 serde_json = { version = "1", features = ["arbitrary_precision"] }
 serde-hex = "0.1.0"
 serde_derive = "1"
-stacks-rpc-client = "1.0.10"
+stacks-rpc-client = "2"
 clarinet-utils = "1.0.0"
 hiro-system-kit = { version = "0.3.1", optional = true }
 # stacks-rpc-client = { version = "1", path = "../../../clarinet/components/stacks-rpc-client" }
 # clarinet-utils = { version = "1", path = "../../../clarinet/components/clarinet-utils" }
 # hiro-system-kit = { version = "0.1.0", path = "../../../clarinet/components/hiro-system-kit" }
-chainhook-types = { version = "1.1.0", path = "../chainhook-types-rs" }
+chainhook-types = { version = "1", path = "../chainhook-types-rs" }
```

Comment:
I find it easier to manage internal dependencies like so, let me know what you think

Reply:
I don't think that the `chainhook-sdk` and `chainhook-types-rs` are mature enough to go with a loose specification. We're not strictly using semver yet, and sometimes just bumping the minor / patch for non backward compatible changes.
The interfaces of the SDK and types are still a bit in flux and being refined, could we please push these (reasonable) changes for later?

Reply:
Rollbacked